src/app/components/core/south/south-service-modal/south-service-modal.component.html

```diff
@@ -46,13 +46,18 @@
                 </a>
               </span>
               <ng-template #staticLabel>
-                <span class="label is-size-6">Applications</span>
+                <span class="label is-size-6 add-application">Applications</span>
```

Comment:
Why did not we have `add-application` class here; different colour for different views? I intentionally left for north and may be we need some typography class instead of this extra (badly named) class and css with opacity?